registry.json

```diff
@@ -1,4 +1,19 @@
 [
+    {
+      "ticker":"KAYA",
+      "poolId":"pool1vfayajkdc0gw876lax5va6ygpfpsd8kz73k4y977emm3clnxlm5",
```

Comment:
Please can you correct the pool id? I believe it is 627a4ecacdc3d0e3fb5fe9a8cee8880a43069ec2f46d5217decef71c